### PR TITLE
Add TestSessionTimeout info in configure.md

### PR DESCRIPTION
**Pull request by Faizan2304**

### CODE REVIEW

File: docs/configure.md [hunk 1]
```diff
@@ -193,6 +197,7 @@ Available elements are:
 | ResultsDirectory  | string | Directory for test run reports. E.g. trx, coverage etc.                                         |
 | SolutionDirectory | string | Working directory for test invocation. Results directory can be relative to this. Used by IDEs. |
 | MaxCpuCount       | int    | Degree of parallelization, spawns `n` test hosts to run tests. Default: 1. Max: Number of cpu cores. |
+| TestSessionTimeout | int   | Testplatform will canceal the test run after it exceeded given TestSessionTimeout and will show the results of tests which ran till that point. |
```
Comment (codito) [inline docs/configure.md]: Please add a note mentioning the version where this supported. E.g. `Version: 15.5+`.

Comment (codito) [inline docs/configure.md]: s/canceal/cancel

Comment (Faizan2304) [inline docs/configure.md]: Added

Comment (Faizan2304) [inline docs/configure.md]: Fixed

**Pull request merged**